From 34bb9eef0ce9349f4f32a0133e695d5f3a1f67b4 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 16 Oct 2023 12:07:43 +0200
Subject: [PATCH 0007/1695] core: arm: add guards to tz_proc_def.h

Adds missing header guards in tz_proc_def.h.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8f51f316ed48553adc8e0b874e5957055b3fab16 core: arm: add guards to tz_proc_def.h]
Change-Id: Ia9c26500812770beda56e0f16655ad3e9f6da7a9
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from cdc44bac56d0d2c08749d5b1900c6ffcb66435c8]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/include/kernel/tz_proc_def.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/arch/arm/include/kernel/tz_proc_def.h b/core/arch/arm/include/kernel/tz_proc_def.h
index 6e9097dfe..2d69b853e 100644
--- a/core/arch/arm/include/kernel/tz_proc_def.h
+++ b/core/arch/arm/include/kernel/tz_proc_def.h
@@ -3,6 +3,9 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
+#ifndef __KERNEL_TZ_PROC_DEF_H
+#define __KERNEL_TZ_PROC_DEF_H
+
 #include <stdint.h>
 
 /*
@@ -89,3 +92,5 @@
 #define LINE_FIELD_OVERFLOW      U(13)
 /* start of way number field offset in way/index format */
 #define WAY_FIELD_OFFSET         U(30)
+
+#endif /*__KERNEL_TZ_PROC_DEF_H*/
-- 
2.47.2

